src/components/discreteSlider/index.stories.mdx

```diff
@@ -0,0 +1,87 @@
+import { useState } from 'react';
+import { ArgsTable, Canvas, Meta, Story } from '@storybook/addon-docs';
+
+import { Center } from '@chakra-ui/react';
+
+import Box from '../box';
+
+import DiscreteSlider from './index';
+
+<Meta
+  title="Components/Filter/Discrete Slider"
+  component={DiscreteSlider}
+  argTypes={{
+    marks: {
+      control: 'object',
+      name: 'marks',
+      description:
+        'An array of marks of generic type T. Each mark is an object that represents a position on the discreteSlider. It has a label, which is a string representing the visual label on the mark, and an arbitrary value that can be of any type.',
+    },
+    value: {
+      control: 'object',
+      name: 'value',
+      description:
+        "The default value where the discreteSlider's thumb will be upon initialization.",
+    },
+    onValueChanged: {
+      control: 'action',
+      name: 'onValueChanged',
+      description:
+        'A function that is called whenever the discreteSlider value changes. It receives the current discreteSlider value as an argument.',
+    },
+    applyIndentation: {
+      control: 'boolean',
+      name: 'applyIndentation',
+      table: {
+        defaultValue: { summary: 'True' },
+      },
+      description:
+        'An optional parameter that lets you define whether the first discreteSlider step should be indented or start right at the beginning of the SliderTrack.',
+    },
+  }}
+/>
+
+# Discrete Slider
+
+DiscreteSlider is a custom component built utilizing the Chakra UI Slider component.
+This slider allows for discreet selections represented by marks, that reflect stepped values.
+This can be useful in scenarios where you want to offer a small selection of options on a slider for a user to pick from.
+
+## Props
+
+<ArgsTable story={'Overview'} />
+
+## Overview
+
+export const Template = (args) => {
+  const marks = [
+    { label: '14 days', value: 1 },
+    { label: '30 days', value: 2 },
+    { label: '60 days', value: 3 },
+    { label: 'Unlimited', value: 4 },
+  ];
+  const [step, setStep] = useState(marks[0].value);
+  return (
+    <Center>
+      <Box width={392}>
+        <DiscreteSlider
+          {...args}
+          marks={marks}
+          value={step}
+          onValueChanged={setStep}
+        />
+      </Box>
+    </Center>
+  );
+};
+
+<Canvas>
+  <Story
+    name="Overview"
+    args={{
+      applyIndentation: true,
+    }}
```

Comment:
Please check if this is redundant now that we've changed the default value.

Reply:
This is more related to storybook itself. By providing this arg we get nice control toggle for controlling boolean value in storybook documentation, which is way more convenient as it reflect changes on component immediately.

Reply:
Makes sense, thank you for the explanation 👍